src/components/reusable/errorBlock/errorBlock.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta, Story, Canvas, Controls } from '@storybook/blocks';
+import * as ErrorStories from './errorBlock.stories.js';
+import alertImg from './redfords/Alert_Redford.svg';
+
+<Meta of={ErrorStories} />
+
+# Error Block
+Error block.
+
+## Usage
+
+### JS Import
+```js
+import '@kyndryl-design-system/shidoka-applications/components/reusable/errorBlock';
+```
+
+### HTML Tag
+```html
+<kyn-error-block></kyn-error-block>
+```
+
+## Importing the error svg
+The following varients of error images/**Redford's** are available in the ```src/components/reusable/errorBlock/redfords```.
```

Comment:
We should store these files in the `common` folder, under a new folder called `assets` or similar. Will also need to update the build/rollup script to make sure the assets get included.

Reply:
done